extension/env/development.env.js

```diff
@@ -1,8 +1,10 @@
-var merge = require('webpack-merge')
-var prodEnv = require('./production.env')
+var merge = require("webpack-merge");
+var prodEnv = require("./production.env");
+var common = require("./common");
 
 module.exports = merge(prodEnv, {
-  NODE_ENV: '"development"',
-  HOST: '"127.0.0.1:8000"',
-  USE_API_MOCK: 'false'
-})
+    NODE_ENV: '"development"',
+    HOST: '"127.0.0.1:8000"',
+    USE_API_MOCK: "false",
+    ...common,
+});
```

Comment:
Add a notice to the readme to indicate this file must be created manually and populated with WSS URL after a make deploy of the aws infra